functions/_fzf_search_history.fish

```diff
@@ -0,0 +1,13 @@
+# originally implemented and transposed from https://github.com/patrickf3139/dotfiles/pull/11
+function _fzf_search_history -d "Search command history using fzf. Replace the commandline with the selected command."
+    # history merge incorporates history changes from other fish sessions
+    history merge
+
+    # Reference https://devhints.io/strftime to understand strftime format symbols
+    if set -l commandWithTs (history --show-time="%m/%e %H:%M:%S | " | fzf --tiebreak=index --query=(commandline))
```

Comment:
Are you sure you don't need `--null`? I'm concerned that if you have multi-line history fzf will get confused and break up the command.

Reply:
Oh really good point, I overlooked that. I'll test it out real quick.